molecule/default/molecule.yml

```diff
@@ -27,6 +27,9 @@ platforms:
 
 provisioner:
   name: ansible
+  config_options:
+    defaults:
+      pipelining: True
```

Comment:
this is the workaround provided in https://docs.ansible.com/ansible/latest/user_guide/become.html#risks-of-becoming-an-unprivileged-user?

if we want this to make everything work we should document it for the user too

env var could be of help in multiple use case
https://docs.ansible.com/ansible/latest/reference_appendices/config.html#ansible-pipelining

vagrant included

Reply:
I saw it mostly as a speedup measure. I actually though it was already in place, but due to molecule using it's own ansible.cfg it slipped off.
Having said that the postgres role requires pipelining for `postgres_user` tasks. No specific action is required by the user as it's explicit in the tasks but some sudo config may prohibit that.
Given postgres role is only provided for convenience and we offer the ability to use an external db for prod env. I think it's fair to document the requirement for the postgres  host.